exercises/practice/acronym/test.sh

```diff
@@ -1,13 +1,9 @@
 #!/usr/bin/env bash
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )/
 SLUG=${1:-$(basename "${SCRIPT_DIR}")}
-COBOLCHECK=./bin/cobolcheck
+COBOLCHECK="${HOME}/cobolcheck/cobolcheck"
 
-WHICH_COBOLCHECK=$(which cobolcheck)
-if [[ $? -eq 0 ]] ; then
-    echo "Found cobolcheck, using $COBOLCHECK"
-    COBOLCHECK=$WHICH_COBOLCHECK
-elif [ ! -f $SCRIPT_DIR/bin/cobolcheck ]; then
```

Comment:
@SimaDovakin I think the remove of this part is what breaks the test runner. The cobol test runner downloads the cobolcheck file to: https://github.com/exercism/cobol-test-runner/blob/main/Dockerfile#L16 

What I would suggest to do is:

- Either revert the WHICH part that was deleted, or
- Change the location of cobolcheck in the test runner (and also update the `test.sh` files in the `tests/*` directories to match what is used here.

Reply:
Hmm... Ok, I'll return "WHICH" part back.